packages/expo/plugins/plugin.ts

```diff
@@ -116,7 +116,7 @@ function buildExpoTargets(
     },
     [options.exportTargetName]: {
       command: `expo export`,
-      options: { cwd: projectRoot },
+      options: { cwd: projectRoot, args: ['--clear'] },
```

Comment:
Wanted to note that this issue also occurs with `expo start`, and presumably other build related commands.

I've added a [comment](https://github.com/nrwl/nx/issues/27392#issuecomment-2311522488) to the issue with additional details and a potential solution

Reply:
i think your solution is better. i will change my pr